Example/SSAppUpdater/Views/ContentView.swift

```diff
@@ -29,6 +30,15 @@ struct ContentView: View {
 
 // MARK: - Private view
 extension ContentView {
+    private func checkForManualMacAppUpdate() {
+        SSAppUpdater.shared.performManualMacAppVersionCheck(
+            url:  "", // Provide server URL
```

Comment:
Add some static variable so that when anyone runs an example get an error and get the idea to need to add a URL here.